src/ui/src/app/page.tsx

```diff
@@ -55,80 +56,112 @@ const Home: React.FC = () => {
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sed
-              condimentum mauris.
+              Uses fine-grained fuzzy matching to assess how closely the blocked
+              candidates compare with the incoming records across several
+              different attributes. Each candidate then receives a Link Score
+              reflecting its quality as a potential match.
             </p>
           </ProcessListItem>
           <ProcessListItem>
-            <ProcessListHeading type="h3">
-              Configure the algorithm
+            <ProcessListHeading type="h3" className="font-sans-md">
+              Pass phase
             </ProcessListHeading>
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sed
-              condimentum mauris.
+              Performs the blocking and evaluation steps again for each
+              combination of fields based on a user-specified number of passes.
+              This lets Record Linker account for missing data and changes over
+              time (such as a person moving and updating their address).
             </p>
           </ProcessListItem>
           <ProcessListItem>
-            <ProcessListHeading type="h3">
-              Test (and adjust as needed)
+            <ProcessListHeading type="h3" className="font-sans-md">
+              Aggregation phase
             </ProcessListHeading>
             <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam sed
-              condimentum mauris.
+              Collects the Link Scores calculated across all passes and sorts
+              the results to determine the most likely patient match.
             </p>
           </ProcessListItem>
         </ProcessList>
+        <CaptionedImage
+          width={792}
+          height={372}
+          src="/images/record-linker-process-min.png"
+          alt="record linker algorithm steps explained"
+          caption={
+            <>
+              Record Linker process diagram &mdash;{" "}
+              <a
+                href="/files/record_linker_process_diagram.pdf"
+                download
+                className="usa-link"
+              >
+                download a diagram
+              </a>{" "}
+              with additional details.
+            </>
+          }
+        />
+        <h2 className="margin-top-8">Where we&apos;re going </h2>
+        <p className="margin-top-1 margin-bottom-4">
+          Record Linker is currently at the product validation stage. Our team
+          is working with users at several public health departments to develop
+          a standalone Master Patient Index that enables patient matching across
+          internal tools (e.g., disease surveillance, vital records, and
+          immunization information systems), thereby improving interoperability
+          between these datasets.
+        </p>
```

Comment:
I know we got the announcement during retro (after you put up this PR) but I'm wondering if we should modify this again.

Reply:
Good catch!